examples/stats/exporter/README.md

```diff
@@ -0,0 +1,89 @@
+# Examples
+
+
+## Stackdriver
+
+The sample uses Application Default Credentials to authenticate with the Stackdriver service.
+
+See https://developers.google.com/identity/protocols/application-default-credentials
+
+You will need to:
+
+1. Create a Google Cloud Platform project
+2. Enable billing
+3. Create a Stackdriver workspace
+4. Create a service account and ensure it is permitted to write Stackdriver metrics
+5. Run the sample
+
+### Create Google Cloud Platform project and enable billing
+
+```
+PROJECT=[[YOUR-PROJECT]]
+BILLING=[[YOUR-BILLING]]
+
+gcloud projects create ${PROJECT}
+
+gcloud beta billing projects link ${PROJECT} \
+--billing-account=${BILLING}
+```
+### Create Stackdriver workspace
+
+```
+google-chrome console.cloud.google.com/monitoring/?project=${PROJECT}
+```
+
+### Create Service Account and set IAM policy
+
+```
+ACCOUNT=[[YOUR-SERVICE-ACCOUNT]]
+ADDRESS=${ACCOUNT}@${PROJECT}.iam.gserviceaccount.com
+
+WORK=[[YOUR-WORKING-DIRECTORY]]
+FILE="${WORK}/${ACCOUNT}.key.json"
+
+gcloud iam service-accounts create ${ACCOUNT} \
+--display-name=${ACCOUNT} \
+--project=${PROJECT}
+
+gcloud projects add-iam-policy-binding ${PROJECT} \
+--member=serviceAccount:${ADDRESS} \
+--role=roles/monitoring.metricWriter
+
+gcloud iam service-accounts keys create ${FILE} \
+--iam-account=${ACCOUNT}@${PROJECT}.iam.gserviceaccount.com \
+--project=${PROJECT}
+```
+
+### Run it
+
+```
```

Comment:
Do you think we should install cmd here?
```
npm install @opencensus/core
npm install @opencensus/exporter-stackdriver
```

Reply:
Fixed by adding a general `package.json` and instructions: 713b4d2fcac33d5a72fdd466f4729402f3acf668

Added Dockerfiles and instructions too.